app/controllers/application_controller.rb

```diff
@@ -63,4 +63,10 @@ def prompt_to_set_up_2fa
   def prompt_to_enter_otp
     redirect_to user_two_factor_authentication_url
   end
+
+  def prompt_to_confirm_mobile(mobile, mobile_taken = false)
+    user_session[:unconfirmed_mobile] = mobile
+    user_session[:unconfirmed_mobile_taken] = mobile_taken
+    redirect_to phone_confirmation_new_path
+  end
```

Comment:
I would put this in a controller concern since it's only used by 2 controllers, whereas putting it in application_controller makes it available to all controllers.